cmd/provenanced/config/manager.go

```diff
@@ -109,34 +109,34 @@
 	return conf, nil
 }
 
-// ExtractTmConfigAndMap from the command context, creates a tendermint config and related string->value map.
-func ExtractTmConfigAndMap(cmd *cobra.Command) (*cmtconfig.Config, FieldValueMap, error) {
-	conf, err := ExtractTmConfig(cmd)
+// ExtractCmtConfigAndMap from the command context, creates a cometbft config and related string->value map.
+func ExtractCmtConfigAndMap(cmd *cobra.Command) (*cmtconfig.Config, FieldValueMap, error) {
+	conf, err := ExtractCmtConfig(cmd)
 	if err != nil {
 		return nil, nil, err
 	}
 	fields := MakeFieldValueMap(conf, true)
-	removeUndesirableTmConfigEntries(fields)
+	removeUndesirableCmtConfigEntries(fields)
 	return conf, fields, nil
 }
 
-func DefaultTmConfig() *cmtconfig.Config {
+func DefaultCmtConfig() *cmtconfig.Config {
 	rv := cmtconfig.DefaultConfig()
 	rv.Consensus.TimeoutCommit = DefaultConsensusTimeoutCommit
 	rv.TxIndex.Indexer = "null"
 	return rv
 }
 
-// removeUndesirableTmConfigEntries deletes some keys from the provided fields map that we don't want included.
+// removeUndesirableCmtConfigEntries deletes some keys from the provided fields map that we don't want included.
 // The provided map is altered during this call. It is also returned from this func.
-// There are several fields in the tendermint config struct that don't correspond to entries in the config files.
+// There are several fields in the cometbft config struct that don't correspond to entries in the config files.
 // None of the "home" keys have entries in the config files:
 // "home", "consensus.home", "mempool.home", "p2p.home", "rpc.home"
 // There are several "p2p.test_" fields that should be ignored too.
 // "p2p.test_dial_fail", "p2p.test_fuzz",
 // "p2p.test_fuzz_config.*" ("maxdelay", "mode", "probdropconn", "probdroprw", "probsleep")
 // This info is accurate in Cosmos SDK 0.43 (on 2021-08-16).
-func removeUndesirableTmConfigEntries(fields FieldValueMap) FieldValueMap {
+func removeUndesirableCmtConfigEntries(fields FieldValueMap) FieldValueMap {
```

Comment:
There's no need to put those strings in variables because they're not needed anywhere else. And having them hard-coded there makes it easier to read and understand exactly what's being looked for.

That being said, I did switch to using `strings.HasSuffix` and `strings.HasPrefix` for the if clause to make it even easier to read.